bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/QvTool.scala

```diff
@@ -22,27 +23,28 @@ import scala.swing.{Frame, Swing}
 class QvTool(ctx: QvContext) extends Frame {
 
   // -- create UI and register frame
-  QvTool.viewerService.map(_.registerView(this))
+  QvTool.viewerService.foreach(_.registerView(this))
   title = s"Queue Visualization - ${ctx.site.displayName}"
   menuBar = new QvToolMenu(this, ctx)
   contents = new QvToolPanel(ctx)
 
   // -- reactions
   reactions += {
     case WindowClosing(e) =>
-      QvTool.viewerService.map(_.unregisterView(this)) // un-register frame when frame is closed
+      QvTool.viewerService.foreach(_.unregisterView(this)) // un-register frame when frame is closed
   }
 
   // -- show the QV tool
   pack
+  Resources.setOTFrameIcon(peer)
   visible = true
 }
 
 /**
  * Make utility services used in QV available throughout the plugin.
  */
 object QvTool {
-  var defaultsFile: File = null
+  var defaultsFile: File = _
```

Comment:
I'm not sure how or where this is used but perhaps we should switch it to an `Option[File]` anyway.